command-expansion-server/sql/commanding/tables/command_dictionary.sql

```diff
@@ -25,3 +26,17 @@ comment on column command_dictionary.version is e''
   'A human-meaningful version qualifier.';
 comment on constraint command_dictionary_natural_key on command_dictionary is e''
   'There an only be one command dictionary of a given version for a given mission.';
+
+create function trigger_command_dictionary_set_updated_at()
```

Comment:
Thanks for making the change! Though I think the trigger prefix should be on the trigger, not on the function.

Reply:
That is what I get for context switch back to AERIE to make the change really quick. Thanks for catching that it should be updated now.